.github/scripts/trigger-workflow.js

```diff
@@ -8,45 +8,39 @@ const { Octokit } = require('@octokit/rest');
 const triggerWorkflow = async () => {
   try {
     const octokit = new Octokit({
-      auth: process.env.REPO_WORKFLOW_TOKEN
+      auth: process.env.REPO_ACCESS_TOKEN,
     });
     const workflowRef = process.env.WORKFLOW_NAME;
     const ref = process.env.BRANCH_NAME;
-    const [owner, repo] = process.env.REPO_NAME
-      ? process.env.REPO_NAME.split('/')
-      : [null, null];
+    const [owner, repo] = process.env.REPO_NAME ? process.env.REPO_NAME.split('/') : [null, null];
 
     // Decode inputs, this MUST be a valid JSON string
-    let inputs = {}
+    let inputs = {};
     const inputsJson = process.env.INPUTS;
-    if(inputsJson) {
-      inputs = JSON.parse(inputsJson)
+    if (inputsJson) {
+      inputs = JSON.parse(inputsJson);
     }
 
     const workflow = await octokit.rest.actions.getWorkflow({
       owner,
       repo,
-      workflow_id: workflowRef
+      workflow_id: workflowRef,
     });
 
-    core.info(`Workflow id is: ${workflow.data.id}`)
+    core.info(`Workflow id is: ${workflow.data.id}`);
 
     const dispatchResp = await octokit.rest.actions.createWorkflowDispatch({
       owner,
       repo,
       workflow_id: workflow.data.id,
       ref,
-      inputs
+      inputs,
     });
-    core.info(`API response status: ${dispatchResp.status} 🚀`)
+    core.info(`API response status: ${dispatchResp.status} 🚀`);
```

Comment:
You need to close the quotes. Also, what's the story behind the emoji?

Reply:
I think the rocket's nose cone is hiding the `

Reply:
Haha, yeah. There is no story :sweat_smile:, I took the emoji from [here](https://github.com/benc-uk/workflow-dispatch/blob/master/src/main.ts#L55).

Reply:
Haha okay. But the rocket would only make sense if the status was positive

Reply:
Yeah, that's true. Updated.